composer.json

```diff
@@ -11,7 +11,7 @@
     ],
     "license": "MIT",
     "require": {
-        "php": "~5.4"
+        "php": ">=5.4"
```

Comment:
another possibility for this would be: `"php": "~5.4 || ~7.0"`.  This would not open up things to 8.0+